entity-management-solr/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>entity-management</artifactId>
+    <groupId>eu.europeana.api</groupId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <groupId>eu.europeana.api</groupId>
+  <artifactId>entity-management-solr</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+
+  <name>entity-management-solr</name>
+  <description>A simple entity-management-solr.</description>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+  
+  	<dependency>
+		<groupId>eu.europeana.api</groupId>
+		<artifactId>entity-management-definitions</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+	</dependency>
+	
+	 <dependency>
+	  	<groupId>org.apache.solr</groupId>
+	  	<artifactId>solr-solrj</artifactId>	  
+	  	<version>${version.solrj}</version>	
+	</dependency>
+	
+	<dependency>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-data-solr</artifactId>
+	</dependency>
+	
+		<dependency>
+			<groupId>eu.europeana.api.commons</groupId>
+			<artifactId>commons-error</artifactId>
+			<version>${api.commons.version}</version>
+		</dependency>
+	
+	<!-- https://mvnrepository.com/artifact/org.json/json -->
+	
+  </dependencies>
+
+  <build>
+   
+  </build>
+
+  <reporting>
```

Comment:
do we need this plugin? If not, please remove

Reply:
Removed.